bip-musig2.mediawiki

```diff
@@ -57,16 +57,13 @@ The MuSig2 variant in this specification stands out by combining all of the foll
 
 == Overview ==
 
-Implementers of the specification must make sure to understand this section thoroughly to avoid subtle mistakes that may lead to catastrophic failure.
+Implementers of this specification must make sure to understand this section thoroughly to avoid subtle mistakes that may lead to catastrophic failure.
 
-This specification is written with a focus on clarity.
-As a result, the specified algorithms are not always optimal in terms of computation and space.
-In particular, some values are recomputed but can be cached in actual implementations (see [[#signing-flow|Signing Flow]]).
```

Comment:
Hm, I wouldn't drop this. It is useful to note that this specification isn't optimal in some regards and implementers can/should cache values. But perhaps the Design section is a better place for this.

Reply:
Oh yes, I wanted to move this, and not drop it. But my plan was to move it to "Algorithm" because it affects just the algorithms. I'll try this.

Reply:
done